### ISSUE TITLE
HTML in emails broken

### ISSUE THREAD

**Issue by aerugo** [loomio/loomio#4568]
In some recent update, HTML in emails broke. See attached screenshot. 

<img width="661" alt="screen shot 2018-03-05 at 12 10 39" src="https://user-images.githubusercontent.com/7785081/36972174-77c0fc66-206e-11e8-87af-24dbf5a5797a.png">

**Comment by gdpelican** [comment]
This has been fixed in the latest master. We'll merge master into stable soon; in the meantime, feel free to track the master image for a bit.

**Comment by aerugo** [comment]
Hmm, are you sure? I just started tracking my recent merge of the master branch, and I still seem to get this problem. Master has started using cable right, while the stable hasn't? I recognised that in the boot-up logs, so I should have the master version running.

<img width="870" alt="screen shot 2018-03-07 at 09 19 09" src="https://user-images.githubusercontent.com/7785081/37081085-a5bfc55e-21e8-11e8-9eda-f48732cc9c18.png">
